seqr/views/apis/samples_api.py

```diff
@@ -11,8 +12,12 @@ def match_sample_ids_to_sample_records(
         project,
         sample_ids,
         sample_type,
+        dataset_type,
+        elasticsearch_index=None,
         max_edit_distance=0,
-        create_sample_records=False):
+        create_sample_records=False,
+        sample_individual_mapping=None,
```

Comment:
would add new params to doc string. Also, not sure how strongly you feel about these names, but would vote for renaming sample_individual_mapping to `sample_id_to_individual_id_mapping`